frontend/dashboard/components/RepoList/FavoriteButton.tsx

```diff
@@ -27,14 +27,20 @@ export const FavoriteButton = ({ repo }: FavoriteButtonProps): React.ReactElemen
     appName: repo.name,
   });
 
-  return (
-    <Button title={title} onClick={handleToggleFav} variant={'tertiary'} icon>
-      {repo.hasStarred ? (
-        <StarFillIcon className={classes.favoriteIcon} />
-      ) : (
-        <StarIcon className={classes.favoriteIcon} />
-      )}
-    </Button>
+  return repo.hasStarred ? (
+    <StudioButton
+      title={title}
+      onClick={handleToggleFav}
+      variant='tertiary'
+      icon={<StarFillIcon className={classes.favoriteIcon} />}
+    />
+  ) : (
+    <StudioButton
+      title={title}
+      onClick={handleToggleFav}
+      variant='tertiary'
+      icon={<StarIcon className={classes.favoriteIcon} />}
+    />
```

Comment:
Maybe we can refactor like this: 

```suggestion
  return (
    <StudioButton
      title={title}
      onClick={handleToggleFav}
      variant='tertiary'
      icon={repo.hasStarred ? <StarFillIcon className={classes.favoriteIcon} /> : <StarIcon className={classes.favoriteIcon} />}
    />
)
```

Reply:
Nice find! I changed it to this to have the same logic for `title` and `icon`:
````suggestion
  const title = t(repo.hasStarred ? 'dashboard.unstar' : 'dashboard.star', {
    appName: repo.name,
  });

  const icon = repo.hasStarred ? (
    <StarFillIcon className={classes.favoriteIcon} />
  ) : (
    <StarIcon className={classes.favoriteIcon} />
  );

  return <StudioButton title={title} onClick={handleToggleFav} variant='tertiary' icon={icon} />;
````

Reply:
Great job! 😄 
Looks even better! 🥳